src/cases/components/constants/allocation.constants.ts

```diff
@@ -0,0 +1,7 @@
+import { RoleAllocationType } from '../../enums';
+
+export const ROLE_ALLOCATION_CONSTANTS = {
+  Exclusion: { allocation: RoleAllocationType.Exclusion },
+  Judiciary: { allocation: RoleAllocationType.Judiciary },
+  LegalOps: { allocation: RoleAllocationType.LegalOps }
+};
```

Comment:
would this best to move to src\work-allocation-2\components, if this is only used in work allocation 2.